File: Backend/Application/DTOs/CriarUsuarioDTO.cs
namespace Backend.Application.DTOs
{
    public class CriarUsuarioDTO
    {
        public required string Nome { get; set; }
        public required string Cpf { get; set; }
        public required string Email { get; set; }
        public required string Senha { get; set; }
        public required string ConfirmarSenha { get; set; }

    }
}

File: frontend/src/features/users/dtos/CriarUsuarioDTO.ts
import type { TipoUsuario } from "../enums/tiposUsuario";

export type CriarUsuarioDTO = {
  nome: string;
  cpf: string;
  email: string;
  senha: string;
  confirmarSenha: string;
  tipoUsuario: TipoUsuario;
};
